spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'cancan/matchers'
```

Comment:
Why are you requiring these matchers here? I don't see them being necessary here...